pkg/events/derive/symbols_loaded_test.go

```diff
@@ -160,14 +161,19 @@ func TestDeriveSharedObjectExportWatchedSymbols(t *testing.T) {
 		},
 	}
 	pid := 1
+	baseLogger := logger.Base()
```

Comment:
Same for these changes, they should be together with symbols_loaded change (and independent from the utils/proc/ns* changes).